engine-rest/engine-rest/pom.xml

```diff
@@ -651,31 +651,15 @@
       <properties>
         <!-- Test with Resteasy version used by the oldest
              Camunda-supported Wildfly/JBoss EAP version.
-             Currently this is JBoss EAP 7.0 (3.0.16.Final) and 7.1 (3.0.24.Final). -->
-        <version.resteasy>3.0.16.Final</version.resteasy>
+             Currently, this is JBoss EAP 7.2. -->
+        <version.resteasy>3.6.1.SP2</version.resteasy>
       </properties>
-      <!-- Resteasy 3.0.X is more similar to Resteasy 2.x, so we need to define separate
-           dependencies for testing compatibility with older Wildfly/JBoss versions.
-           We can drop the 'dependencies' and 'build' sections below once we drop support for
-           Resteasy 3.0.X -->
       <dependencies>
         <dependency>
           <groupId>org.jboss.spec.javax.ws.rs</groupId>
-          <artifactId>jboss-jaxrs-api_1.1_spec</artifactId>
+          <artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
```

Comment:
Can we bump the artifact here as well:
https://github.com/camunda/camunda-bpm-platform/blob/83b2ef8a31967cc7578c055564943490ea037bac/engine-rest/engine-rest/pom.xml#L285-L295

Reply:
The classpath exclusion is not needed anymore. When this configuration was introduced, `jboss-jaxrs-api_1.1_spec` was defined in the default profile. The dependency was excluded to avoid having different versions on the classpath during test execution.